docs/guides/how_to_guides/spare_parts/how_to_create_custom_expectations.rst

```diff
@@ -1,4 +1,4 @@
-.. _how_to_guides__creating_and_editing_expectations__how_to_create_custom_expectations:
+.. _how_to_guides__creating_and_editing_expectations__how_to_create_custom_expectations_sp:
```

Comment:
just curious. what is the role of the `_sp` at the end?

Reply:
There was a name collision, so I added "sp" to denote this as "spare parts".